### DISCUSSION TITLE
Vercel REST API changes regarding to the response object of add domain to a project Api in vercel

### DISCUSSION THREAD

**Discussion by MihirBhardwaj07** [vercel/community#4008] [Bug Report]
### Page to Investigate

https://vercel.com/docs/rest-api/endpoints

### Steps to Reproduce

call the /v10/projects/{idOrName}/domains for domain add to a project the response is supposed to return a verification object list but now its not returning that object ......

**Comment by amyegan** [comment]
Hey @MihirBhardwaj07. If `verified` is `true`, then there won't be any `verification` items to show.
